src/Symfony/Component/Form/Tests/DependencyInjection/FormPassTest.php

```diff
@@ -166,7 +206,13 @@ public function testAddTaggedFormTypeExtensionWithoutExtendedTypeAttribute()
             ->setPublic(true)
             ->addTag('form.type_extension');
 
-        $container->compile();
+        try {
+            $container->compile();
+
+            $this->fail();
+        } catch (InvalidArgumentException $e) {
+            $this->assertSame('"form.type_extension" tagged services must have the extended type configured using the extended_type/extended-type attribute, none was configured for the "my.type_extension" service.', $e->getMessage());
```

Comment:
why not keeping `@expectedException` and `@expectedExceptionMessage` ?

Reply:
`@expectedDeprecation` would not be evaluated then